pyproject.toml

```diff
@@ -34,17 +34,17 @@ repository = "https://github.com/laike9m/pdir2"
 [project.optional-dependencies]
 [tool.pdm.dev-dependencies]
 dev = [
-    "pytest~=6.2",
-    "tox~=3.24",
-    "tox-pdm~=0.3",
-    "ptpython~=3.0",
-    "bpython~=0.21",
-    "ipython~=7.16",
-    "flake8~=3.9",
-    "pytest-mypy~=0.8",
-    "pandas~=1.1",
-    "hypothesis~=6.21",
-    "tox-gh-actions~=2.8",
+    "pytest==6.2.*",
+    "tox==3.24.*",
+    "tox-pdm==0.3.*",
+    "ptpython==3.0.*",
+    "bpython==0.21.*",
+    "ipython==7.16.*",
+    "flake8==3.9.*",
+    "pytest-mypy==0.8.*",
+    "pandas==1.3.5",
```

Comment:
some of the versions are not working anymore, so I have to manually fix them by locking the version to a working one. it's painful TT